solr/packaging/powershell-tests/Zk.Tests.ps1

```diff
@@ -42,6 +42,81 @@ BeforeAll {
   $script:ZK_PORT = if ($env:ZK_PORT) { $env:ZK_PORT } else { "9983" }
   $script:SOLR_PORT = if ($env:SOLR_PORT) { $env:SOLR_PORT } else { "8983" }
 
+  # Check if Solr is already running
+  $solrAlreadyRunning = $false
+  try {
+    $response = Invoke-WebRequest -Uri "http://localhost:$SOLR_PORT/solr/" -UseBasicParsing -TimeoutSec 2 -ErrorAction SilentlyContinue
+    if ($response.StatusCode -eq 200) {
+      $solrAlreadyRunning = $true
+      Write-Host "Solr is already running on port $SOLR_PORT"
+    }
+  } catch {
+    $solrAlreadyRunning = $false
+  }
```

Comment:
A bit disaapointing that while the BATS `setup_file()` function is a tw-liner, Pester needs 100 lines for the same? Why? Guess we could pull some of the verbose check-if-solr-running and start-solr out in Util scripts?

Reply:
Moving this out makes sense, yeah. This can be done in one go together with the other topics I believe.